opa-filter-autoconfigure/build.gradle.kts

```diff
@@ -0,0 +1,19 @@
+dependencies {
```

Comment:
Seeing these declarations here I'm no longer as convinced of this way of doing it - I had to hop through 3 files just to know which version a dependency was at.

Reply:
It becomes painless when seen in an IDE, although I fully understand your concerns. Initially, versions and dependencies were living in the same file, maybe that's the way to go? Versions need to be separated from `build.gradle` anyway, so that they're synced across modules.

Reply:
From my perspective is better to split `build.gradle` into smaller ones for each subproject. They're just more readable then. :+1:

Reply:
Not suggesting you change anything here, just thinking out loud here. Go with what feels best.

Reply:
> From my perspective is better to split `build.gradle` into smaller ones for each subproject. They're just more readable then.

They will stay separate, no worries. My suggestion was to merge file with dependency definitions and their versions. I'll do that, so you all can see if you like it.

Reply:
@lkadalski @anderseknert I've merged `Versions.kt` and `Dependencies.kt`. Let me know if you're still not sure we should go that way, I can put that info in root `build.gradle.kts`.

Reply:
If you like it the way it is you should leave it like it is - it's been an interesting discussion but ultimately this will be easy and non-breaking to change later should we want to.

Reply:
I'll create a task for making this decision :)

Reply:
Done: GH-2